packages/framework/composer.json

```diff
@@ -50,7 +50,7 @@
         "arvenil/ninja-mutex": "^0.4.1",
         "bmatzner/jquery-bundle": "^2.2.2",
         "bmatzner/jquery-ui-bundle": "^1.10.3",
-        "commerceguys/intl": "0.7.4",
+        "commerceguys/intl": "^0.7.4",
```

Comment:
Why was it 0.7.4 earlier? How was it resolved?

Reply:
i will add it into commit message after approval of changes